From e9fbd1458b8744b468ed22d11949a749c8282d9a Mon Sep 17 00:00:00 2001
Message-ID: <e9fbd1458b8744b468ed22d11949a749c8282d9a.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:21:04 +0000
Subject: [PATCH v3 47/66] gnu: go-github-com-aws-aws-sdk-go-v2-config: Move to
 (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-aws-aws-sdk-go-v2-config): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Ie775eff088ac4ad0327e2479a63f677ac9c75da2
---
 gnu/packages/golang-web.scm | 12 ++++++++++++
 gnu/packages/golang.scm     | 11 -----------
 2 files changed, 12 insertions(+), 11 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 7ecdcbd8b5..fde9a89ea3 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -158,6 +158,18 @@ (define-public go-github-com-aws-aws-sdk-go-v2
      "Package sdk is the official AWS SDK v2 for the Go programming language.")
     (license license:asl2.0)))
 
+(define-public go-github-com-aws-aws-sdk-go-v2-config
+  (package
+    (inherit go-github-com-aws-aws-sdk-go-v2)
+    (name "go-github-com-aws-aws-sdk-go-v2-config")
+    (version "1.18.5")
+    (arguments
+     '(#:import-path "github.com/aws/aws-sdk-go-v2/config"
+       #:unpack-path "github.com/aws/aws-sdk-go-v2"))
+    (propagated-inputs
+     (list go-github-com-google-go-cmp-cmp
+           go-github-com-aws-smithy-go))))
+
 (define-public go-github-com-aws-smithy-go
   (package
     (name "go-github-com-aws-smithy-go")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index d631973702..98753c17b4 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -11300,17 +11300,6 @@ (define-public go-github-com-dvsekhvalnov-jose2go
     (home-page "https://github.com/dvsekhvalnov/jose2go")
     (license license:expat)))
 
-(define-public go-github-com-aws-aws-sdk-go-v2-config
-  (package
-    (inherit go-github-com-aws-aws-sdk-go-v2)
-    (name "go-github-com-aws-aws-sdk-go-v2-config")
-    (version "1.18.5")
-    (arguments
-     '(#:import-path "github.com/aws/aws-sdk-go-v2/config"
-       #:unpack-path "github.com/aws/aws-sdk-go-v2"))
-    (propagated-inputs (list go-github-com-google-go-cmp-cmp
-                             go-github-com-aws-smithy-go))))
-
 (define-public go-github-com-aws-aws-sdk-go-v2-service-iam
   (package
     (inherit go-github-com-aws-aws-sdk-go-v2)
-- 
2.41.0

